src/Components/PageTemplate/index.js

```diff
@@ -46,11 +58,13 @@ const PageTemplate = (props) => {
                 </div>
             }
             <div className="col-12 col-sm-4 col-md-4 col-lg-4 gx-0">
-                <EmailList emails={emails} />
+                <EmailList emails={emails} emailToBeDisplayedId={emailToBeDisplayedId} setEmailToBeDisplayedId={setEmailToBeDisplayedId} emailDisplayVisible={emailDisplayVisible} setEmailDisplayVisible={setEmailDisplayVisible} screenWidth={screenWidth}/>
             </div>
-            <div className="d-none d-sm-block col-sm-6 col-6 col-md-5 col-lg-7 gx-0">
-                <EmailDisplay />
+            {emailDisplayVisible &&
+            <div className="emailDisplayContainer d-sm-block col-sm-6 col-6 col-md-5 col-lg-7 gx-0">
```

Comment:
should be col-md-8 ? check ipad view, the column does not take up the right width. currently using 9 of the 12 space at medium screen size